[docs] Update templates for starting the tutorial and an Expo Router app (#28766)
expo/expo: docs/pages/tutorial/follow-up.mdx

### Navigation

> We had one screen in the tutorial app. Most apps have multiple screens. [Expo Router](/router/introduction/) allows you to add new screens to your app by adding additional files to your project. Get started with Expo Router already configured by creating your next app with the default template:

Reviewer: ```suggestion
We had one screen in the tutorial app. Most apps have multiple screens. [Expo Router](/router/introduction/) allows you to add new screens to your app by adding additional files to your project. To create a new project with Expo Router library already installed and configured, see [Create a project](/get-started/create-a-project/).
```

Let's add a link to the Get Started section instead of specifying the command here. Since the tutorial targets new users, this is a good point for them to go explore Setting their environment (Expo Go vs Development builds) in the next guide after create a project.

> <Terminal cmd={['$ npx create-expo-app@latest']} />

Reviewer: ```suggestion
```
